### ISSUE TITLE
`0.4.7` won't start on Powershell/Windows

### ISSUE THREAD

**Issue by eeintech** [sparkmicro/Ki-nTree#69]
![image](https://user-images.githubusercontent.com/4020546/139495697-9ff9190e-8692-477f-a91c-a63bc2610f5c.png)

The `/` might be the problem!

**Comment by deepthought-64** [comment]
This seems to be not only a window issue. I tried starting it today under Linux (Arch) and got the same error.
```
> python3 ./kintree
Traceback (most recent call last):
  File "/home/fabi/.local/bin/./kintree", line 8, in <module>
    sys.exit(kintree_gui.main())
  File "/home/fabi/.local/lib/python3.9/site-packages/kintree/kintree_gui.py", line 913, in main
    with open('images/logo.png', 'rb') as imagefile:
FileNotFoundError: [Errno 2] No such file or directory: 'images/logo.png'
```

**Comment by eeintech** [comment]
Thanks for the report @deepthought-64 I hope to fix this issue soon!

**Comment by eeintech** [comment]
@deepthought-64 Should be fixed in `0.4.8` :wink:
